R/bundle.R

```diff
@@ -15,9 +15,20 @@ bundleAppDir <- function(appDir, appFiles, appPrimaryDoc = NULL) {
         file == appPrimaryDoc) {
       to <- file.path(bundleDir, "app.R")
     }
+
     if (!file.exists(dirname(to)))
       dir.create(dirname(to), recursive = TRUE)
     file.copy(from, to)
+
+    #ensure .Rprofile doesn't call packrat/init.R
+    if (basename(to) == ".Rprofile") {
+      origRprofile <- readLines(to)
+      newRprofile <- gsub( 'source(\"packrat/init.R\")',
+                           '#source(\"packrat/init.R\")',
```

Comment:
Could you add a comment to the `.Rprofile` here indicating why the `source()` line was commented out, so that it's clear what's going on if someone opens the bundle? e.g.

    # Modified by rsconnect package $VERSION on $DATE:
    # Packrat initialization disabled in published application
    # source ...